server/database/dbconnection.js

```diff
@@ -0,0 +1,17 @@
+const { Pool } = require("pg");
+require("env2")("./config.env");
+
+let { DEV_DB_URL } = process.env;
+// if (process.env.NODE_ENV === 'test') {
```

Comment:
this is fine to be uncommented

Reply:
I've checked the database, and everything is there. I'm currently writing the tests.